### PR TITLE
allow single column span

### LINKED ISSUE #8: Add a single column option
Currently, tables automatically span two columns.

### PR DESCRIPTION
**Pull request by gvonkoss**
Closes #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,11 +66,11 @@ The grid does not require the sidebar to maintain its layout.
 #### Layout Base
 The markup below ↓ will generate the ascii grid above ↑.  
 
-The main content section will constrain its _immediate_ children to its first column, with the exception of `table`s and `aside`s.
-
-
+The main content section will constrain its _immediate_ children to its first column, with the exception of `table`s and `aside`s.  
 If you want an element to span the full width of the main content area, you can apply the `o-layout__main--full-span` class to that element to achieve that effect.
 
+_Note: `o-layout` styles tables to span two columns automatically. If you want a table to stick to a single column, you can apply_ `o-layout__main--single-span`, _instead. Every other component is styled to fit a single column._
```
Comment (notlee) [inline README.md]: "Every other component is styled to fit a single column" I'm not sure I understand, do you mean non-table elements span a single column?

Comment (gvonkoss) [reply README.md]: yeah that should say element not component.... :retreat:

File: demos/src/content/3-party-table.mustache [hunk 2]
```diff
@@ -13,7 +13,7 @@
 </ol>
 
 <h2 id="this-is-a-table">This is a table</h2>
-<table class="o-table o-table o-table--row-stripes" data-o-component="o-table">
+<table class="o-table o-table o-table--row-stripes o-layout__main--single-span" data-o-component="o-table">
```
Comment (notlee) [inline demos/src/content/3-party-table.mustache]: BEM whoops `o-layout__main__single-span`

Comment (gvonkoss) [reply demos/src/content/3-party-table.mustache]: that would make `o-layout__main--full-span` wrong also.

Comment (notlee) [reply demos/src/content/3-party-table.mustache]: Good spot, could we update both but keep `o-layout__main--full-span` also for backwards compatibilty?

**Pull request merged**